### DISCUSSION TITLE
Does the video setting encoder 'Get from OBS' require additional encoding?

### DISCUSSION THREAD

**Discussion by Toanl31** [sorayuki/obs-multi-rtmp#404] [Q&A] [answered]
Does the video setting encoder 'Get from OBS' require additional encoding?

Example: Stream on Twitch and YouTube. 

YouTube is the main encoder on OBS settings via h264 NVENC
Twitch is the RTMP 'Get from OBS' setting. 

Will Twitch through RTMP require a second encoding session via h.264 NVENC or only the Mbit bandwidth for uploading?
In other words 1 Encode -> Multiple RTMP via additional internet bandwidth. If have gigabit, could do whatever amount granted there is enough bandwidth.

**Comment by sorayuki** [comment, marked as answer]
No. You will see "share with: OBS" if you select "Get from OBS"
________________________________
From: Toanl31 ***@***.***>
Sent: Wednesday, February 14, 2024 1:22:28 AM
To: sorayuki/obs-multi-rtmp ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [sorayuki/obs-multi-rtmp] Does the video setting encoder 'Get from OBS' require additional encoding? (Discussion #404)


Does the video setting encoder 'Get from OBS' require additional encoding?

Example: Stream on Twitch and YouTube.

YouTube is the main encoder on OBS settings via h264 NVENC
Twitch is the RTMP 'Get from OBS' setting.

Will Twitch through RTMP require a second encoding session via h.264 NVENC or only the Mbit bandwidth for uploading?
In other words 1 Encode -> Multiple RTMP via additional internet bandwidth. If have gigabit, could do whatever amount granted there is enough bandwidth.

―

**Reply by Toanl31** [reply]
Thank you.
